lib/rubocop/cop/naming/memoized_instance_variable_name.rb

```diff
@@ -55,7 +55,7 @@ def on_def(node)
           msg = format(
             MSG,
             var: ivar_assign.children.first.to_s,
-            suggested_var: method_name.to_s.chomp('?'),
+            suggested_var: method_name.to_s.delete('!?'),
```

Comment:
Method name can contain `!` and `?` at the end of the name only. So we can use `String#delete` for this purpose.